native/core/src/execution/jni_api.rs

```diff
@@ -281,14 +279,13 @@ fn parse_memory_pool_config(
     memory_pool_type: String,
     memory_limit: i64,
     memory_limit_per_task: i64,
-    memory_fraction: f64,
 ) -> CometResult<MemoryPoolConfig> {
     let memory_pool_config = if use_unified_memory_manager {
         MemoryPoolConfig::new(MemoryPoolType::Unified, 0)
     } else {
         // Use the memory pool from DF
-        let pool_size = (memory_limit as f64 * memory_fraction) as usize;
-        let pool_size_per_task = (memory_limit_per_task as f64 * memory_fraction) as usize;
```

Comment:
The `memory_fraction` is already applied in JVM side.

Reply:
If memory_fraction is not used here, we can remove the configuration option `COMET_EXEC_MEMORY_FRACTION (spark.comet.exec.memoryFraction)` since it is not used in the code base.

My understanding is that `spark.comet.exec.memoryFraction` is still useful. As per the documentation, it makes DataFusion not using up all the reserved native memory, since some of the allocations are untracked, and this is also a compensation of inaccurate memory usage estimations.